test/integration/utils/sn-table.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-await-in-loop */
+import { Locator, Page } from '@playwright/test';
+import { Collection, Key } from '../types';
+
+type KeyPress = {
+  key: Key;
+  times: number;
+};
+
+const selectors: Collection = {
+  chart: '#chart-container',
+  text: (text: string): string => `text=${text}`,
+  focused: 'th[tabindex="0"], td[tabindex="0"]',
+};
+
+const textToDelete: string = '\nPress space to sort on this column';
+
+class SnTable {
+  readonly page: Page;
+
+  public selectors: Collection;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.selectors = selectors;
+  }
+
+  async getSelectedColumn(parent?: string): Promise<Collection> {
+    const loc: Locator = this.page.locator(selectors.header.selected(parent));
+    await loc.isVisible();
+    const innerText: string = await this.getInnerTextAndReplace(loc, textToDelete, '');
+    const sort: string = (await loc.getAttribute('aria-sort')) as string;
+    return { innerText, sort };
+  }
+
+  async getTextFromLocator(parent: Locator | string, child: Locator | string): Promise<string[]> {
+    let parentLocator: Locator;
+    let childLocator: Locator;
+
+    if (typeof parent === 'string') parentLocator = this.page.locator(parent);
```

Comment:
Curly brackets here please. It know the linter does not complain (it should..) but it makes it harder to read just to save a few lines of code.